### ISSUE TITLE
Odd release title when releasing workspace

### ISSUE THREAD

**Issue by domoritz** [axodotdev/cargo-dist#98]
I set up a workspace and the release title was "v0.7.2v0.7.2v0.7.2v0.7.2". I had to manually fix it to be just `v0.7.2`. 

See https://github.com/domoritz/arrow-tools/releases/tag/v0.7.2

**Comment by Gankra** [comment]
I'm guessing this is #69

**Comment by domoritz** [comment]
Yes! I have a specific case where this breaks if that's useful.

**Comment by Gankra** [comment]
Do you always plan to do unified releases of everything in lockstep, or do you want more fine-grain control?

Also how do you want things to be merged?

(Feel free to reply in the other issue)

**Comment by domoritz** [comment]
I am okay with making coordinated releases since I usually update all tools together. But I can imagine sometimes I want to only push a fix to one tool. 

I ran into https://github.com/axodotdev/cargo-dist/issues/95 when I just tried the default settings of cargo release (without `shared-version` and `tag-name`.

FWIW, here is what previous releases looked like in an individual repo: https://github.com/domoritz/csv2parquet/releases/tag/v0.6.1
